examples/cloud-operations/multi-cluster-mesh-gke-fleet-api/templates/vars.yaml.tpl

```diff
@@ -0,0 +1,8 @@
+istio_version: ${istio_version}
+clusters:
+%{ for cluster in clusters ~}
+  - ${cluster}    
+%{ endfor ~}  
+region: ${region}  
+service_account_email: ${service_account_email}
+project_id: ${project_id}
```

Comment:
you could use a [auto.tfvars.json](https://www.terraform.io/language/values/variables#variable-definitions-tfvars-files) file built with `jsonencode`